RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -165,7 +196,11 @@ struct SubscriptionDetailsView: View {
                     .foregroundColor(Color.gray)
                     .padding([.horizontal, .bottom])
             }
-        }
+        }.padding()
+            .padding(.horizontal)
+            .background(Color.white)
```

Comment:
Just checking but does this work well in dark mode?

Reply:
Good point, it doesn't haha. I changed them to `UIColor`s which adapt to the color scheme: https://github.com/RevenueCat/purchases-ios/pull/4072/commits/461a28b59b69b304f50a52c019474fe157144eac